tests/Knp/Menu/Tests/Iterator/CurrentItemFilterIteratorTest.php

```diff
@@ -17,7 +17,7 @@ public function testSimpleFiltering(): void
 
         $names = [];
         // FilterIterator expects an Iterator implementation explicitly, not an IteratorAggregate.
-        $iterator = new CurrentItemFilterIterator($this->menu->getIterator(), new Matcher());
+        $iterator = new CurrentItemFilterIterator(new \ArrayIterator($this->menu->getChildren()), new Matcher());
```

Comment:
I'd rather use `new IteratorIterator($this->menu)`, which is the built-in way in PHP to wrap a Traversable into an Iterator.